internal/fs/local_file_test.go

```diff
@@ -788,7 +781,7 @@ func (t *LocalFileTest) TestReadSymlinkForDeletedLocalFile() {
 	err = os.Remove(filePath)
 	AssertEq(nil, err)
 	err = t.closeLocalFile(&t.f1)
-	t.validateIOError(err)
+	AssertEq(nil, err)
```

Comment:
Not seeing this change in this PR - https://github.com/GoogleCloudPlatform/gcsfuse/pull/1358

Reply:
Symlink changes were added in a follow PR on top of this.

https://github.com/GoogleCloudPlatform/gcsfuse/pull/1333